npm/vite-dev-server/src/plugins/sourcemap.ts

```diff
@@ -15,7 +15,11 @@ export const CypressSourcemap = (
     enforce: 'post',
     transform (code, id, options?) {
       try {
-        if (/\.js$/i.test(id) && !/\/\/# sourceMappingURL=/i.test(code)) {
+        // Remove query parameters from the id. This is necessary because some files
+        // have a cache buster query parameter (e.g. `?v=12345`)
+        const queryParameterLessId = id.split('?')[0]
+
+        if (/\.js$/i.test(queryParameterLessId) && !/\/\/# sourceMappingURL=data/i.test(code)) {
```

Comment:
Suggest adding a comment here stating that we are looking for files without a sourcemap or without a non-inline sourcemap.

Reply:
Added: [`91bf502` (#30606)](https://github.com/cypress-io/cypress/pull/30606/commits/91bf5021a1fcb18a7621f04e83a8129d4cdafcc8)